suites/message/nested.go

```diff
@@ -307,6 +320,34 @@ func MessageTest_NestedSends(t *testing.T, factory state.Factories) {
 		td.AssertHead(builtin.InitActorAddr, prevHead)  // Init state unchanged.
 	})
 
+	t.Run("fail insufficient funds for transfer in inner send", func(t *testing.T) {
+		puppetBalance := big.Zero()
+		td := builder.WithActorState(drivers.ActorState{
```

Comment:
instantiate the puppet actor as needed for testing to keep stateroots consistent.

Reply:
Great. When this drivers.ActorState construction appears more than once please extract to a function.